.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
```

Comment:
@coderabbitai list all PRs for milestone 1.19.0 that has a bug label.

Reply:
@coderabbitai list all merged PRs for milestone 1.19.0 that has a bug label.

Reply:
There should be more than 7 PRs. There should be 50 of these.

Reply:
Give me the exhaustive list

Reply:
@coderabbitai provide me the exhaustive list of all merged PRs for milestone 1.19.0 that has a bug label.

Reply:
@coderabbitai provide me the exhaustive list of all merged PRs for milestone 1.19.0 that has a enhancement label.

Reply:
Give me all contributors in this PR in a paragraph sentence and ordered alphabetically.

Reply:
Give me all contributors for milestone 1.19.0 in a sentence and ordered alphabetically.

Reply:
Give me all contributors for milestone 1.19.0

Reply:
Give me all milestone 1.19.0 PR assignees

Reply:
Find all PRs that have milestone "1.19.0", from those PRs list the assignees.

Reply:
List authors that have committed in this PR

Reply:
Lisy authors that are in the PR description links

Reply:
Look at PR description's list of linked pull requests and find the authors